chart/kubeapps/templates/kubeapps-frontend-config.yaml

```diff
@@ -53,6 +53,7 @@ data:
       }
 
       location /api/tiller-deploy {
+        proxy_read_timeout 10m;
```

Comment:
I am not sure what you mean, can you elaborate? Why wouldn't it?

Reply:
Well when accessing Kubeapps over the port-forward, you are directly accessing this nginx which means all this configuration will apply as expected. However, when you use an Ingress, you access another Nginx (or other, e.g. traefik), before accessing this one. If that first Nginx/other does not have this read timeout, is this going to be an issue?

Reply:
Ah, yes, I understand. Yes, if the first Nginx doesn't increase the timeout that one will close the connection. There are annotations to increase that timeout for the Nginx ingress controller but that may not work for other solutions like traefik. I am not sure how much do we need to get into that?

I will add the timeout for the Nginx ingress though.

Reply:
@andresmgot are we 100% sure this is needed? Before, you had this timeout set in the wrong nginx and it seemed like you didn't notice a problem. Is that because it took less than the default 60s? How can we reproduce the problem this setting solves?

We should also add a comment as to why this is needed, if it is.

Reply:
> Is that because it took less than the default 60s? How can we reproduce the problem this setting solves?

Yes, it is because of that. I reproduced the issue without this line (the dashboard returns a 502 bad gateway because the connection is closed).